WcaOnRails/app/models/country.rb

```diff
@@ -38,6 +47,6 @@ def self.find_by_iso2(iso2)
       # "Tunisia" even if the country displayed is actually "Tunisie"...)
       [e.name_in(l), e.id]
```

Comment:
Not this PR, but

1. Can we use `locale` and `country` instead of `l` and `e`?

2. Isn't "search will behave weirdly" a euphemism for "search will no longer work"?